src/test/storage/any_segment_iterable_test.cpp

```diff
@@ -27,15 +28,17 @@ class AnySegmentIterableTest : public BaseTestWithParam<SegmentEncodingSpec> {
                            {ChunkID{0}, ChunkOffset{1}}, {ChunkID{0}, ChunkOffset{1}}, {ChunkID{0}, ChunkOffset{5}}};
     position_filter = std::make_shared<PosList>(std::move(row_ids));
     position_filter->guarantee_single_chunk();
+
+    std::copy_if(all_segment_encoding_specs.begin(), all_segment_encoding_specs.end(),
+                 std::back_inserter(int_supporting_segment_encodings), [](SegmentEncodingSpec spec) {
+                   return encoding_supports_data_type(spec.encoding_type, DataType::Int);
```

Comment:
I find a simple loop easier to read.

Reply:
Hmpf ... did not check the result of formatting. The one liner was much nicer than this. :(

Will change it back to a for loop.